tests/rip7560/conftest.py

```diff
@@ -72,17 +72,21 @@ def tx_7560(wallet_contract):
 
 @pytest.fixture(scope="session")
 def nonce_manager(w3):
+    artifact_path = "/../../@rip7560/artifacts/contracts/predeploys/NonceManager.sol/NonceManager.json"
+    return contract_from_artifact(w3, artifact_path, CommandLineArgs.nonce_manager)
+
+
+@pytest.fixture(scope="session")
+def stake_manager(w3):
+    artifact_path = "/../../@rip7560/artifacts/contracts/predeploys/StakeManager.sol/StakeManager.json"
```

Comment:
for tests we usually compile contracts from sol. why here depend on precompiled json ? it requires an ugly "prepare" step

Reply:
Because these are predeployed contracts...